npm package fails on 0.1.11

It is installed successfully but on usage, it fails with

```
/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/ts-node/src/index.ts:513
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
app/conn/mssql.ts:6:28 - error TS2306: File '/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/@pimeys/connection-string/connection_string.d.ts' is not a module.

6 import { JdbcString } from '@pimeys/connection-string'
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~

    at createTSError (/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/ts-node/src/index.ts:513:12)
    at reportTSError (/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/ts-node/src/index.ts:517:19)
    at getOutput (/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/ts-node/src/index.ts:752:36)
    at Object.compile (/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/ts-node/src/index.ts:968:32)
    at Module.m._compile (/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/ts-node/src/index.ts:1056:42)
    at Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/divyendusingh/Documents/prisma/introspection-ci/node_modules/ts-node/src/index.ts:1059:12)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
error Command failed with exit code 1.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,7 +17,7 @@ authors = [
 [package.metadata.wasm-pack.profile.release]
 wasm-opt = ["-Oz", "--enable-mutable-globals"]
 
-[target.'wasm32-unknown-unknown'.dependencies]
+[target.'cfg(target_arch = "wasm32")'.dependencies]
 wasm-bindgen = "0.2"
 
 [dev-dependencies]

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -30,7 +30,7 @@ mod jdbc;
 mod utils;
 
 /// The WASM representations of the connection string structs.
-#[cfg(target = "wasm32-unknown-unknown")]
+#[cfg(target_arch = "wasm32")]
 pub mod wasm;
 
 pub use ado::AdoNetString;

